### ISSUE TITLE
can you write code to have this cookie?

### ISSUE THREAD

**Issue by khanhyepp77** [Rdimo/Hazard-Token-Grabber-V2#225]
![image](https://user-images.githubusercontent.com/100086076/164506134-9a1e0a1c-b714-4fe2-9efa-bf24162414c4.png)

**Comment by Rdimo** [comment]
alot of people have wanted me to have some sort of cookie formatting or output it as json so you can import it easier, since alot of people want this I might aswell add it, will make the formatting customizable too

**Comment by khanhyepp77** [comment]
thanks, i will wait for you

**Comment by aligollez** [comment]
we waiting

**Comment by Rdimo** [comment]
right, forgot about this. Could someone contact me saying how it should be formatted?

**Comment by Rdimo** [comment]
> 

doesn't really tell me anything, give context and what parts is which
for example the `TRUE / FALSE`, what does that indicate?

**Comment by aligollez** [comment]
https://github.com/zhenghuijie/NetscapeHttpCookie/blob/master/http_cookie_2_netscape_cookie.py

**Comment by Rdimo** [comment]
ah alr, ye ill consider adding this formatting

**Comment by aligollez** [comment]
> ah alr, ye ill consider adding this formatting

I'm looking, maybe I can add

**Comment by aligollez** [comment]
I tried a lot of stealers, I think yours is the most stable.
I also do 100% fud, that's the only thing missing because I can't import cookies :)

**Comment by Rdimo** [comment]
> I tried a lot of stealers, I think yours is the most stable. I also do 100% fud, that's the only thing missing because I can't import cookies :)

ye ill look into the formatting, ty

**Comment by LuckyDucky123** [comment]
> I tried a lot of stealers, I think yours is the most stable. I also do 100% fud, that's the only thing missing because I can't import cookies :)

mind helping me with detections and shit? i'll pay you if needed c:
- lunarz#4351 (add me)

**Comment by aligollez** [comment]
I leave the cookies as .db without converting them to text, then I found a solution by exporting json.
